lib/mi_bridges/driver/pregnancy_information_page.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+module MiBridges
+  class Driver
+    class PregnancyInformationPage < BasePage
+      TITLE = "Pregnancy Information"
+
+      delegate :members, to: :snap_application
+
+      def setup; end
+
+      def fill_in_required_fields
+        if no_one_pregnant?
+          click_no_one
+        else
+          members.each do |member|
+            check_in_section(
+              "starPregnancyAllPrograms",
+              member.new_mom?,
+              first_name_section(member),
+            )
+          end
+        end
+      end
+
+      def continue
+        click_on "Next"
+      end
+
+      private
+
+      def click_no_one
+        check_in_section "starPregnancyAllPrograms"
```

Comment:
I had to read this a couple of times before I was sure it was working as expected (it is!) 

Wonder if we should have a separate method called `click_no_one` in `BasePage` that just takes a single arg (for clarity) eg:

```
def check_no_one(section)
  check_in_section(section)
end
```